velox/dwio/parquet/reader/PageReader.cpp

```diff
@@ -363,6 +366,32 @@ void PageReader::prepareDictionary(const PageHeader& pageHeader) {
           // We start from the end to allow in-place expansion.
           values[i] = parquetValues[i];
         }
+      } else if (
+          type_->type()->isTimestamp() && parquetType == thrift::Type::INT64) {
+        VELOX_CHECK(type_->logicalType_.has_value());
+        auto logicalType = type_->logicalType_.value();
+        if (logicalType.__isset.TIMESTAMP) {
+          VELOX_CHECK(
+              logicalType.TIMESTAMP.isAdjustedToUTC,
```

Comment:
Should this be VELOX_NYI?

Reply:
Btw, what do you want to do if isAdjustedToUTC is false?

Reply:
I haven't thought about it much. My perspective is from Spark, I think it writes in UTC adjusted. But once this framework is working it shouldn't be much effort to add that support in a followup PR.